examples/certs/gcp/gcp.ext

```diff
@@ -4,4 +4,5 @@ keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
 subjectAltName = @alt_names
 
 [alt_names]
+DNS.1 = aggregator.oakexamples.dev
 IP.1 = 35.246.87.178
```

Comment:
do we still need the IP address here?

Reply:
I think we don't really need it. But it is not harmful to keep it. 
Or is it actually needed for certain scenarios @conradgrobler?

Reply:
It is not really needed, but in future if the WebPKI crate does start supporting IP addresses in certs, it could be used for connections as well.